engine/runtime-compiler/src/main/scala/org/enso/compiler/pass/PassManager.scala

```diff
@@ -231,6 +247,51 @@ class PassManager(
     ix - totalLength == indexOfPassInGroup
   }
 
+  /** Validates consistency between the IR accessible via `moduleContext` and `ir`.
+    * There is no way how to enforce this consistency statically.
```

Comment:
```suggestion
    * There is no way to enforce this consistency statically.
```

Reply:
Typo fixed in 85ab82f3ce9891445b4cd4ccddbbdff7361abf51